templates/zerver/api/changelog.md

```diff
@@ -20,6 +20,16 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 6.0
 
+**Feature level 133**
+
+* [`PATCH /streams/{stream_id}`](/api/update-stream): The endpoint now
+  returns a 404 response for a request to make a public stream with
```

Comment:
Is it really a 404 ("Not Found")? I'd have expected 400 ("Bad Request"). If it is 404, that's a surprising quirk.

Either way, though, I think the changelog should just say it returns an error.

Reply:
Yeah this is `400` only, my bad. Also updated this to just say returns an error.